docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 This directory contains all of the user documentation for NGINX Ingress Controller, as well as the requirements for building and publishing the documentation.
 
-Documentation is written in Markdown, built using [Hugo](https://gohugo.io) with [nginx-hugo-theme](https://github.com/nginxinc/nginx-hugo-theme), then deployed with [Netlify](https://www.netlify.com/).
+Documentation is written in Markdown, built using [Hugo](https://gohugo.io) with [nginx-hugo-theme](https://github.com/nginxinc/nginx-hugo-theme).
```

Comment:
@nginx-jack is it possible to include docs on how to access preview builds?

Reply:
Of course! We might add the docs to the [shared action itself,](https://github.com/nginxinc/docs-actions) and link out since a few repos will need it.

Reply:
https://github.com/nginxinc/docs-actions/pull/20